### PR TITLE
Added message attributes for Domain Events

### PR DESCRIPTION
**Pull request by LunaBulnesCMP**

### CODE REVIEW

File: src/Cmp/Queues/Domain/Event/DomainEvent.php [hunk 1]
```diff
@@ -252,6 +284,7 @@ public function jsonSerialize()
         return [
             'origin'        => $this->origin,
             'name'          => $this->name,
+            'context'       => $this->context,
```
Comment (joangoat) [inline src/Cmp/Queues/Domain/Event/DomainEvent.php]: you dont want to add attributes here?

File: src/Cmp/Queues/Domain/Event/JSONDomainEventFactory.php [hunk 2]
```diff
@@ -22,7 +22,7 @@ public function create($json)
             throw new InvalidJSONDomainEventException("String is not valid JSON");
         }
 
-        if (!isset($domainEventArray['origin'], $domainEventArray['name'], $domainEventArray['version'], $domainEventArray['occurredOn'], $domainEventArray['body'])) {
```
Comment (joangoat) [inline src/Cmp/Queues/Domain/Event/JSONDomainEventFactory.php]: why this change?

Comment (LunaBulnesCMP) [reply src/Cmp/Queues/Domain/Event/JSONDomainEventFactory.php]: because if you check the class DomainEvent, body is optional. So this is actually a bug

File: src/Cmp/Queues/Domain/Event/DomainEvent.php (hunk 1 above)
Comment (LunaBulnesCMP) [reply src/Cmp/Queues/Domain/Event/DomainEvent.php]: No, attributes are, in fact, meta data, and we are only serialising the message in here.

We don't want a field in the message with the actual meta data of the message.
